workflows/mag.nf

```diff
@@ -688,8 +688,10 @@ workflow MAG {
                 }
         }
 
-        ch_versions = ch_versions.mix(BINNING_PREPARATION.out.bowtie2_version.first())
-        ch_versions = ch_versions.mix(BINNING.out.versions)
+        if ( !params.skip_binning || params.ancient_dna ) {
```

Comment:
If we skip binning but enable ancient DNA - will this fail as BINNING.out doesn't exist?

Reply:
Ah yeah, sorry I shouldn't be doing two pipelines at once (I thought this wouldve been a easier fix than I thought 😅)

I'll come back to this tomorrow morning in my normal mag Dev time

Thanks for the rapid feedback though, it's appreciated!

Reply:
No worries, it's just me procrastinating on what I should be doing... 😅

Reply:
OK indeed- it tripped me up the binning prep version mixing wasn't in the same place (same with binning) as the actual invocation of the module!

Reply:
Should be good now: binning prep versions mixed in when executed, and binning is always mixed in as it's already in the binning activiation if/else statement